.github/workflows/slack-notify/action.yml

```diff
@@ -25,22 +25,16 @@ runs:
         aws-secret-access-key: ${{ inputs.aws-secret-access-key }}
         aws-region: us-gov-west-1
 
-    - name: Get Slack app token
-      uses: marvinpinto/action-inject-ssm-secrets@v1.2.1
-      with:
-        ssm_parameter: /devops/github_actions_slack_socket_token
-        env_variable_name: SLACK_APP_TOKEN
-
     - name: Get Slack bot token
       uses: marvinpinto/action-inject-ssm-secrets@v1.2.1
       with:
         ssm_parameter: /devops/github_actions_slack_bot_user_token
         env_variable_name: SLACK_BOT_TOKEN
 
     - name: Notify Slack
-      uses: department-of-veterans-affairs/vsp-github-actions/slack-socket@v1.1.0
+      uses: slackapi/slack-github-action@v1.17.0
+      env:
+        SLACK_BOT_TOKEN: ${{ env.SLACK_BOT_TOKEN }}
       with:
-        slack_app_token: ${{ env.SLACK_APP_TOKEN }}
-        slack_bot_token: ${{ env.SLACK_BOT_TOKEN }}
-        attachments: ${{ inputs.attachments }}
-        channel_id: ${{ inputs.channel_id }}
+        payload: ${{ inputs.attachments }}
```

Comment:
There has been a couple of instance because its treating as a JSON, it will sometimes give syntax error. 

Changing the input from `attachments` to `payload` fixed the issue in `content-build`

Reply:
This was mainly seen in `self-hosted` runners